src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -462,13 +462,6 @@ public static DetailAST parseWithComments(FileContents contents)
     }
 
     @Override
-    public void destroy() {
```

Comment:
Check tests are failing, problem that super implementation is empty so it will do no effect on test.